demos/connector/neo4j/directed-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -25,7 +25,7 @@
    "source": [
```

Comment:
<blockquote>Note that this should be avoided for larger graphs.</blockquote><p> </p><p>Maybe "Note that this should be avoided for graphs where the node features are large" or something, since it's fine if most of the size is in the edges?</p><p> </p><p>(Also applies to undirected.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>